layouts/shortcodes/sidebar-link.html

```diff
@@ -37,6 +38,18 @@
         >Add to Calendar</span
       >
     </button>
+  {{ else if $dateExpired }}
```

Comment:
Can't we just test if today is after the date?

Reply:
`time "2023-08-01"` turns a date string into time object. Then you can compare it with the `now` time object. Something like

```
{{ $dateObj := time $date }}
{{ if now.After $dateObj }}
 ```

https://pkg.go.dev/time#Time.After documents the methods on a time object.

Reply:
hugo can't seem to parse the date from the shortcode. 

`{{ $date := "2023-08-01" }}` works, but `{{ $date := .Get "date" }}` does not.

Reply:
I did a quick test and I ran into the problem that the build fails if `time $date` is "", but I think it should be okay if it's behind an `if $date` statement.

Reply:
Opened a sub-PR.